Tests/test_peep_pip.py

```diff
@@ -0,0 +1,255 @@
+import unittest
+import math
+
```

Comment:
Lots of magic numbers in these tests, would think that a few more comments would clear things up.